internal/pkg/mount/v2/pseudo.go

```diff
@@ -48,7 +49,7 @@ func PseudoSubMountPoints() Points {
 		)
 	}
 
-	if _, err := os.Stat("/sys/fs/selinux"); err == nil {
+	if _, err := os.Stat("/sys/fs/selinux"); err == nil && selinux.IsEnabled() {
```

Comment:
I guess we can then drop the `/sys/fs/selinux` check then?

Reply:
I thought we did the `/sys/fs/selinux` only appeared when `selinux=1` was set

Reply:
Did we add that check for cases like when apparmor is enabled and selinux is then disabled? Then perhaps yes, we can drop that check

Reply:
> Did we add that check for cases like when apparmor is enabled and selinux is then disabled?

I believe so